plansys2_pddl_parser/src/plansys2_pddl_parser/Expression.cpp

```diff
@@ -63,34 +71,90 @@ Expression * createExpression( Stringreader & f, TokenStruct< std::string > & ts
 		f.next();
 		std::string s = f.getToken();
 		if ( s == "+" || s == "-" || s == "*" || s == "/" ) {
```

Comment:
Should we also check for these: "=", ">=", ">", "<=", "<" ?

Reply:
I'm not sure since here we are building expressions (e.g. `(= (* (+ a b) d)))`) and it is not possible to mix equality within expressions. The relational operators are handled above if I'm not mistaken.